src/Microsoft.Sbom.Api/Workflows/SbomRedactionWorkflow.cs

```diff
@@ -57,7 +57,7 @@ public virtual async Task<bool> RunAsync()
                 var validationDetails = await validatedSbom.GetValidationResults();
                 if (validationDetails.Status != FormatValidationStatus.Valid)
                 {
-                    throw new InvalidDataException($"Failed to validate {sbomPath}:\n{string.Join('\n', validationDetails.Errors)}");
+                    throw new InvalidDataException($"Failed to validate {sbomPath}:\n{string.Join('\n', validationDetails.Errors)} Redaction is only supported for SPDX 2.2 currently, please provide a valid SPDX 2.2 SBOM.");
```

Comment:
Grammar--let's use a period instead of a comma here
```suggestion
                    throw new InvalidDataException($"Failed to validate {sbomPath}:\n{string.Join('\n', validationDetails.Errors)} Redaction is only supported for SPDX 2.2 currently. Please provide a valid SPDX 2.2 SBOM.");
```